tox.ini

```diff
@@ -86,7 +86,12 @@ basepython = python3
 skipsdist = true
 skip_install = true
 commands =
-    pytest -rA -vvvv \
+    pytest -rA -vvvv --export-traces \
         --confcutdir=tests/integration \
         {posargs:tests/integration}
-passenv = KRB5CCNAME, REQUESTS_CA_BUNDLE, KRB5_CLIENT_KTNAME, CACHITO_TEST_CERT, CACHITO_TEST_KEY, JOB_NAME
+passenv = KRB5CCNAME,REQUESTS_CA_BUNDLE,KRB5_CLIENT_KTNAME,CACHITO_TEST_CERT\
+          CACHITO_TEST_KEY,JOB_NAME
+setenv =
+    OTEL_RESOURCE_ATTRIBUTES=service.name=cachito-integration-tests
+    OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
+
```

Comment:
I think this file now has two trailing whitelines.

Reply:
And we probably need to re-expose port 4317 https://github.com/containerbuildsystem/cachito/pull/918#discussion_r1358350953